test/e2e/serial/tests/resource_accounting.go

```diff
@@ -85,11 +85,15 @@ var _ = Describe("[serial][disruptive][scheduler][resacct] numaresources workloa
 
 		// Note that this test, being part of "serial", expects NO OTHER POD being scheduled
 		// in between, so we consider this information current and valid when the It()s run.
+
+		By("waiting for the NRT data to settle")
```

Comment:
I'm still unsure we need this here, because 1. BeforeEach() do not do any padding. 2. teardown from prev test will check this.

Reply:
Very fair. Let me tinker with it, and likely remove.